### ISSUE TITLE
[Bug Report] Missing null check in clearableCallback for v-text-field

### ISSUE THREAD

**Issue by nclemeur** [vuetifyjs/vuetify#7059]
### Versions and Environment
**Vuetify:** 1.5.13
**Vue:** 2.6.10
**Browsers:** Firefox 66.0
**OS:** Windows 10

### Steps to reproduce
I can't really reproduce it outside my project, wich use a readonly but clearable v-textarea (value is populated outside the text-area). But anyway, looking at the code, 
```javascript
clearableCallback () {
      this.internalValue = null
      this.$nextTick(() => this.$refs.input.focus())
},
```
while just above we have:
```javascript
blur () {
      this.$refs.input ? this.$refs.input.blur() : this.onBlur()
},
```
so I would think a check for null on this.$refs.input is also required in `clearableCallback `

### Expected Behavior
No console error

### Actual Behavior
Get an error in the console:
```
vue.esm.js?a026:1897 TypeError: Cannot read property 'focus' of undefined
    at VueComponent.eval (vuetify.js?ce5b:16172)
    at Array.eval (vue.esm.js?a026:1989)
    at flushCallbacks (vue.esm.js?a026:1915)
```

### Reproduction Link
<a href="https://codepen.io/nclemeur/pen/KYGaLo?editors=1111" target="_blank">https://codepen.io/nclemeur/pen/KYGaLo?editors=1111</a>

**Comment by nclemeur** [comment]
Sorry should have made more clear that the core referenced above is in:

vuetify/packages/vuetify/src/components/VTextField/VTextField.js

**Comment by nclemeur** [comment]
I have edited the issue as I have managed to reproduce the problem in a codepen

**Comment by jacekkarczmarczyk** [comment]
fixed by f9139eceb42caf43e07593612133e132fe8e4389

**Comment by everhardt** [comment]
Would it be possible to also fix this bug in the 1.5 branch?

**Comment by everhardt** [comment]
Nevermind, I wrote a temporary hack myself (where the autocomplete is referenced as `autocomplete`):

```js
      if (this.$refs.autocomplete !== undefined) {
        // this.$refs.autocomplete.clearableCallback();

        // hack to fix https://github.com/vuetifyjs/vuetify/issues/7059
        // can be dropped when migrated to Vuetify 2
        this.$refs.autocomplete.internalSearch = undefined;
        this.$refs.autocomplete.setValue(undefined);
        this.$nextTick(() => {
          if (this.$refs.autocomplete !== undefined
            && this.$refs.autocomplete.$refs.input !== undefined) {
            this.$refs.autocomplete.$refs.input.focus();
          }
        });
      }
```
